packages/site/package.json

```diff
@@ -33,20 +29,22 @@
 		"@rollup/plugin-json": "^4.1.0",
 		"@rollup/plugin-node-resolve": "^13.0.5",
 		"@rollup/plugin-replace": "^3.0.0",
+		"@sveltejs/kit": "next",
```

Comment:
Since this dependency wasn't pinned, it updated to a more recent stable version of sveltekit (which uses `+page`, or `+layout.svelte` and others for page rendering instead of direct svelte component names). This essentially breaks the whole site - either pin this back to an older version or re-update the site.

Reply:
There's a specific version in the lockfile, so it doesn't need to be pinned